src/query/sql/src/planner/binder/stage.rs

```diff
@@ -91,6 +92,17 @@ impl BindContext {
                 };
                 scalar = ScalarExpr::ConstantExpr(expr);
             }
+
+            // check invalid ScalarExpr
+            if matches!(
+                &scalar,
+                ScalarExpr::WindowFunction(_) | ScalarExpr::AggregateFunction(_)
```

Comment:
Now insert value support udf. If I write a udf that contains a windowfunc or agg func, Is it illegal?